modules/manage/pages/kubernetes/security/authentication/k-user-controller.adoc

```diff
@@ -0,0 +1,274 @@
+
+When you create ACLs with the User resource, the specified ACLs are applied only to the user defined in the `metadata.name` field. For example, if you create ACLs for a user named `data-consumer`, those ACLs apply only to that user. Other users in the Redpanda cluster are not affected by these ACLs.
+
+This example shows how to manage only the ACLs for an existing user in the Redpanda cluster.
+
+.`new-acl.yaml`
+[,yaml,indent=0]
+----
+include::manage:example$kubernetes/user-crds.feature[tags=manage-authz-only-manifest,indent=0]
+----
+
+[[both]]
+=== Create a new user and assign ACLs
+
+- *Use case*: You want to manage both user credentials and ACLs within the same resource.
+- *What happens when deleted*: Both the user and the associated ACLs are removed.
+
+This example shows how to manage both authentication and ACLs for a user within the same User resource.
+
+.`new-user-and-acl.yaml`
+[source,yaml,indent=0]
+----
+# In this example manifest, the user "full-user" is created and managed for both authentication and authorization.
+# The user is granted both read and write access to the topic critical-topic.
+apiVersion: cluster.redpanda.com/v1alpha2
+kind: User
+metadata:
+  name: full-user
+spec:
+  cluster:
+    clusterRef:
+      name: sasl
+  authentication:
+    type: scram-sha-512
+    password:
+      valueFrom:
+        secretKeyRef:
+          name: full-user-secret
+          key: password
+  authorization:
+    acls:
+      - type: allow
+        resource:
+          type: topic
+          name: critical-topic
+          patternType: literal
+        operations: [Read,Write]
+----
+
+== Configuration best practices
+
+The following sections provide guidance on setting up user authentication, managing secrets, and defining ACLs within your Kubernetes environment. These recommendations ensure proper user management while minimizing manual interventions and preventing potential security issues. By following these best practices, you can ensure that user access and permissions are correctly configured and maintained across your Redpanda cluster.
+
+You can find all configuration options for the User resource in the xref:reference:k-crd.adoc#k8s-api-github-com-redpanda-data-redpanda-operator-operator-api-redpanda-v1alpha2-user[CRD reference].
+
+=== Choose a username
+
+The `metadata.name` field in the User resource is used to specify the username. Keep in mind the following best practices when choosing a username:
+
+- Uniqueness: Ensure that each user has a unique name to avoid conflicts. The username must be unique within the Redpanda cluster.
+- Descriptive: Choose a name that identifies the purpose or role of the user. For example, use names like `app-consumer` or `admin-user`.
+- Stable: Avoid changing usernames frequently. Usernames are tied to authentication and authorization rules (ACLs). Renaming a user involves deleting and recreating the user.
```

Comment:
```suggestion
- *Unique*: Ensure each user has a unique name to avoid conflicts. The username must be unique within the Redpanda cluster.
- *Descriptive*: Choose a name that identifies the purpose or role of the user. For example, use names like `app-consumer` or `admin-user`.
- *Stable*: Avoid changing usernames frequently. Usernames are tied to authentication and authorization rules (ACLs). Renaming a user involves deleting and recreating the user.
```